views/templates/Lists/index.tpl

```diff
@@ -23,7 +23,7 @@
             <h6><?= _dh($list['label']) ?></h6>
         </header>
         <div class="container-content" id="list-elements_<?= $list['uri'] ?>">
-            <ol>
+            <ol data-max-items="<?= $listElementsLimit >= 0 ? $listElementsLimit : '';?>">
```

Comment:
Does it make sense to have something that is not a number here? Aren't we externalising ambiguous behavious with this if?

Reply:
Yes, maybe this is overly defensive programming.

The controller always provides this value, and the method that generates it on the PHP side is already type hinted as an int. I'll change it to directly echo the value.

Reply:
Indeed, it is not necessary in this case, otherwise every thing you get from backend you would validate before send to the frond end. We have to trust in the information at some point. In this case we have full control the this should be a number **always**.